src/nip46/utils/request-response.ts

```
@@ -6,12 +6,34 @@
- * Uses a combination of timestamp and random characters
+ * Generate a cryptographically secure random request ID
+ * Uses crypto.randomBytes in Node.js or crypto.getRandomValues in browsers
  */
 export function generateRequestId(): string {
-  const randomChars = Math.random().toString(36).substring(2, 12);
-  return randomChars;
+  // Use Node.js crypto if available
+  if (typeof process !== 'undefined' && process.versions && process.versions.node) {
+    try {
+      // Dynamic import to avoid require statement
+      const crypto = eval('require')('crypto');
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)